lib/build.gradle.kts

```diff
@@ -70,22 +70,7 @@ val buildCrypto = task<Exec>("BuildCrypto") {
 }
 
 val compileContracts = task<Exec>("compileContracts") {
-    val compileScriptPath = "${project.projectDir}/src/test/resources/compileContracts.sh"
-
-    val sourceDirectoryV0 = "${project.projectDir}/src/test/resources/src_v0"
-    val sourceDirectoryV1 = "${project.projectDir}/src/test/resources/src_v1"
-
-    val compileDirectoryV0 = "${project.projectDir}/src/test/resources/compiled_v0"
-    val compileDirectoryV1 = "${project.projectDir}/src/test/resources/compiled_v1"
-
-    inputs.file(compileScriptPath)
-    inputs.dir(sourceDirectoryV0)
-    inputs.dir(sourceDirectoryV1)
-
-    outputs.dir(compileDirectoryV0)
-    outputs.dir(compileDirectoryV1)
-
-    commandLine(compileScriptPath)
+    commandLine("${project.projectDir}/src/test/resources/compileContracts.sh")
```

Comment:
Compiling contracts now is almost instant, so I don't think any local caching is relevant.